src/OrderFormAcceptanceTests.Tests/Features/PrintOrder.feature

```diff
@@ -0,0 +1,12 @@
+﻿Feature: Print Order
+
```

Comment:
would you mind including the As a/I want/So that statement from the story?

> As a Buyer
I want to download a copy of my Order Summary
So that I can include it in a contract